scripts/d4xx_to_mavlink.py

```diff
@@ -600,14 +582,30 @@ def get_local_ip():
         local_ip_address = socket.gethostbyname(socket.gethostname())
     return local_ip_address
 
+
+
+
 ######################################################
 ##  Main code starts here                           ##
 ######################################################
 
-print("INFO: Connecting to vehicle.")
-while (not vehicle_connect()):
-    pass
-print("INFO: Vehicle connected.")
+progress("INFO: pyrealsense2 version: %s" % str(rs.__version__))
```

Comment:
It's greating having the pyrealsense version displayed here so this is a good addition I think.  I'm less sure about the change from "Connecting to Vehicle" to "Starting MAVLink connection" and "STarting MAVLink thread".  Remember this is going to be seen by regular people and I'm not sure many of them know what MAVLink is and probably most don't know what a "thread" is.  Let's keep it simple.

Reply:
I've moved this back to "INFO: Starting Vehicle communications"